tests/test_dark.py

```diff
@@ -65,6 +65,7 @@ def setUpClass(cls):
         config.doWidenSaturationTrails = False
         config.doBrighterFatter = False
         config.doDefect = False
+        config.doSaturation = True
```

Comment:
Isn't this already set to true on line 53?

Reply:
Yes. SO I'm confused by the failing test from the other day. Maybe there was a confusion. I'll double-check. We might not need this PR then.